index.js

```diff
@@ -1,14 +1,285 @@
+    b.style.filter = "blur(2px)";
+    c.style.filter = "blur(2px)";
+    body.style.overflowY = "hidden";
+
+    let title = document.querySelector(".title-one");
+    title.innerHTML = `<h2>${project2.name}</h2>`;
+
+    const canopy = document.querySelector(".canopy");
+    canopy.innerHTML = `           
+ <span class="can-pop"> CANOPY </span>
+<img class="dot" src="./images/Counter.png" alt="" />
+<span class="side-pop"> BACK END DEV </span>
+<img class="dot" src="./images/Counter.png" alt="" />
+<span class="side-pop"> 2015 </span>
+`;
+
+    const theImage = document.querySelector(".pop-websnap");
+    theImage.innerHTML = `
+<img  src="${project2.featuredImage}" alt="Project screenshot" />
+`;
+
+    const paratext = document.querySelector(".pop-text");
+    paratext.innerHTML = `
+<p> ${project2.description} </p>`;
+
+    const theList = document.querySelector(".links-pop");
+    theList.innerHTML = `
+<ul>
+<li class='link'>html</li>
+<li class='link'>css</li>
+<li class='link'>javaScript</li>
+</ul>
+<ul>
+<li class='link hide_ul'>html</li>
+<li class='link hide_ul'>css</li>
+<li class='link hide_ul'>javaScript</li>
+</ul>
+<hr>
+<div class='popup-a' >
+<a href="${project1.linkToLive}" class="pro-pop focusable"> See Live <img class="btn_icon" src="./images/share.png" alt="icon"> </a>
+<a href="${project1.linkToSource}" class="pro-pop focusable"> See Source <img class="btn_icon" src="./images/github_purple.png" alt="icon"> </a>
+`;
+  } else {
+    project.style.display = "none";
+    b.style.filter = "none";
+    c.style.filter = "none";
+    body.style.overflowY = "scroll";
+  }
+}
+
+function popup_project4() {
+  const project = document.getElementById("popup");
+
+  if (project.style.display === "none") {
+    project.style.display = "flex";
+    project.style.overflowY = "scroll";
+    b.style.filter = "blur(2px)";
+    c.style.filter = "blur(2px)";
+    body.style.overflowY = "hidden";
+
+    let title = document.querySelector(".title-one");
+    title.innerHTML = `<h2>${project4.name}</h2>`;
+
+    const canopy = document.querySelector(".canopy");
+    canopy.innerHTML = `           
+ <span class="can-pop"> CANOPY </span>
+<img class="dot" src="./images/Counter.png" alt="" />
+<span class="side-pop"> BACK END DEV </span>
+<img class="dot" src="./images/Counter.png" alt="" />
+<span class="side-pop"> 2015 </span>
+`;
+
+    const theImage = document.querySelector(".pop-websnap");
+    theImage.innerHTML = `
+<img  src="${project4.featuredImage}" alt="Project screenshot" />
+`;
+
+    const paratext = document.querySelector(".pop-text");
+    paratext.innerHTML = `
+<p> ${project4.description} </p>`;
+
+    const theList = document.querySelector(".links-pop");
+    theList.innerHTML = `
+<ul>
+<li class='link'>html</li>
+<li class='link'>css</li>
+<li class='link'>javaScript</li>
+</ul>
+<ul>
+<li class='link hide_ul'>html</li>
+<li class='link hide_ul'>css</li>
+<li class='link hide_ul'>javaScript</li>
+</ul>
```

Comment:
- [ ] You are repeating the same DOM for all the projects. Instead of creating a function for each project to render DOM, I would recommend creating one function and using map() or forEach() methods to loop through the projects array and render their Information.